### PR TITLE
docs: add TSDoc annotations to time functions and properties

### PR DESCRIPTION
**Pull request by deodad**
## Motivation

Add hints for developers about what format different timestamp values are in.

## Change Summary

- add TSDoc comments to places in the public API that deal with time

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [x] The title of this PR adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [x] The PR has been tagged with the appropriate change type label(s) (i.e. documentation, feature, bugfix, or chore)

### CODE REVIEW

File: packages/js/src/types.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export type MessageData<TBody = MessageBody, TType = flatbuffers.MessageType> =
   flatbuffer: flatbuffers.MessageData;
   body: TBody;
   type: TType;
+  /** unix milliseconds timestamp */
   timestamp: number;
```
Comment (adaddeo) [inline packages/js/src/types.ts:28]: having this be the Farcaster time probably makes more sense than unix milliseconds

@pfletcherhill wyt?

**Review by pfletcherhill — COMMENTED** [review]
My assumption was that the deserialized timestamp should be the same format as the input to the builder. That way a user of the package doesn't necessarily need to know what farcaster time is. Am I missing something?

**Comment by adaddeo** [comment]
> My assumption was that the deserialized timestamp should be the same format as the input to the builder. That way a user of the package doesn't necessarily need to know what farcaster time is. Am I missing something?

My thinking was the `timestamp` value of `MessageData` being in the [protocol specified format](https://github.com/farcasterxyz/protocol#timestamps) is the most straightforward. There is a Merkle usage that assumes this is the case, which doesn't seem unreasonable given nothing says otherwise.

Fair point about the builder though. Either way seems fine as long was it's well-documented. Let me know which you prefer and I'll update the TSDoc comments accordingly.

**Comment by adaddeo** [comment]
Sounds good. Updated the PR to just add TSDoc annotations specifying the timestamp format.

**Pull request merged**